planet/http.py

```diff
@@ -414,12 +415,13 @@ async def stream(
         finally:
             await response.aclose()
 
-    def client(self, name: str, base_url: Optional[str] = None) -> object:
+    def client(self,
+               name: Literal['data', 'orders', 'subscriptions'],
+               base_url: Optional[str] = None) -> object:
         """Get a client by its module name.
 
         Parameters:
-            name: the name of the client module: data, orders, or
-                subscriptions.
+            name: one of 'data', 'orders', or 'subscriptions'.
```

Comment:
This is pretty nice, I think. A user is told that they should pass 'data', 'orders', or 'subscription' as the argument.

![Screenshot from 2023-02-27 10-44-36](https://user-images.githubusercontent.com/33697/221642406-6ab32a79-44d8-4252-99a8-8cdd53abe2c0.png)

Reply:
This typing hint IS nice! This would be great to use this for all of the other arguments that have a set number of 'valid' entries, we already have them defined either at the top of the client module or in the specs module.